### PR TITLE
Add oidc-wiremock to CI Security2 category

### PR DESCRIPTION
**Pull request by sberyozkin**
I've also move `elytron-resteasy` to the `HTTP` category where all other resteasy tests are run

### CODE REVIEW

**Review by machi1990 — APPROVED** [review]
LGTM. Let's see what CI says or if there will be a need to increase the timeouts.

**Comment by sberyozkin** [comment]
@machi1990 So it failed, though I've confirmed it works in the native image for me, hmm... need to find the logs somehow :-)

**Comment by sberyozkin** [comment]
Right, so it is
```
java.io.IOException: Failed to bind to /0.0.0.0:8180
2020-10-20T11:37:21.4834171Z Caused by: java.net.BindException: Address already in use
```
since Keycloak is listening there as well, elytron oauth2 tests run on the default 8080 port, so I'll try it

File: integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakTestResource.java [hunk 1]
```diff
@@ -22,24 +21,23 @@
...
 
-        WireMock.configureFor(8180);
         WireMock.stubFor(
                 get(urlEqualTo("/auth/realms/quarkus/.well-known/openid-configuration"))
                         .willReturn(aResponse()
                                 .withHeader("Content-Type", MediaType.APPLICATION_JSON)
                                 .withBody("{\n" +
-                                        "    \"authorization_endpoint\": \"http://localhost:8180/authenticate\",\n" +
-                                        "    \"end_session_endpoint\": \"http://localhost:8180/logout\",\n" +
+                                        "    \"authorization_endpoint\": \"http://localhost:8080/authenticate\",\n" +
```
Comment (gastaldi) [inline integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakTestResource.java:32]: Another option would be to use `server.getOptions().portNumber()`

Comment (sberyozkin) [reply integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakTestResource.java]: @gastaldi @machi1990 it also has `server.baseUrl()` so I've used it and also removed the redundant discovery values which are not needed for the mock testing; it should be very easy for the users to mock the bearer token authentication; code flow would have something more similar to the original, CC @cemnura

**Review by gastaldi — APPROVED** [review]

**Pull request merged**